README.md

```diff
@@ -1,50 +1,38 @@
-# Mentorship Client
+# Cross-platform client for Mentorship System
 
-Cross-platform client for [Systers&#x27; mentorship system](https://github.com/systers/mentorship-backend),
-written in [Flutter](https://flutter.dev/).
+Mentorship System is an application that allows women in tech to mentor each other, on career development topics, through 1:1 relations for a certain period of time.
 
-![demo image](https://i.imgur.com/Xbg7Ty3.png)
+This is the client app for the [Mentorship System backend](https://github.com/anitab-org/mentorship-backend). It's written in the [Flutter](https://flutter.dev) framework.
 
-### How was it created?
+> Work in progress
 
-I really liked [Systers](https://github.com/systers)' Mentorship System, wanted to try Flutter and had some free time during Christmas.
+<img width="764" alt="App running on Android and iOS devices" src="https://i.imgur.com/Xbg7Ty3.png">
 
-### Rationale
+### Purpose
 
-Mentorship System aims to be as inclusive as possible for everyone. It should apply
-to user's mobile operating system, too :)
+AnitaB.org aims to be as inclusive as possible for everyone. It should apply
+to user's mobile operating system, too :) That's why we decided to use Flutter
+to deliver a high-quality app for everybody – Android, iOS, and web users.
 
-### Work status
+From the technical point of view, having one codebase for all platforms will make
+adding new featuresand bug fixes much faster.
 
-Application is a high-fidelity copy of the Android version. It has all essential
-features implemented, though some may not work perfectly.
+## Contributing 
 
-**What works?**
-Almost everything.
+Please read our [Contributing guidelines](https://github.com/anitab-org/mentorship-android/blob/develop/.github/CONTRIBUTING.md), [Code of Conduct](http://systers.io/code-of-conduct) and [Reporting Guidelines](http://systers.io/reporting-guidelines).
```

Comment:
This copy-paste works 🤷‍♀️ but you are pointing to code of conduct and reporting guidelines from systers.io, instead of the files within the repository. and the contribution guidelines are pointing to Android repository.
@bartekpacia I'll just merge and create a first timers issue :)

Reply:
Ah, okay then. It was 3:30 AM when I was doing it so sorry for that.

Reply:
No problem :P You can focus on other tasks from this project! @bartekpacia this change was still valid, all links were valid as they led to valid documentation!
Great thing about open source, is that we can use this little enhancement and bring more people from the community to the project ;)